src/praisonai-agents/mcp-sse-direct-client.py

```
@@ -0,0 +1,120 @@
+# python mcp-sse-direct-client.py http://0.0.0.0:8080/sse
+import asyncio
+import json
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)